### PR TITLE
Update dependency from plugnet to latest stable polkadot

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Updated plugnet's dependency to polkadot and ensured build works fine and all test cases works well..

### CODE REVIEW

**Review by jordy25519 — APPROVED** [review]
LGTM

**Comment by bengooo** [comment]
Discussed with @KarishmaBothara , I would suggest on-holding this PR to be merged, as

- The purpose of the PR was going to release a new version of wallet which could be tested in cennznet, if that is the case, we could apply `yarn workplace` locally rather than making a release; and also, there is no urgent requirement on it as a hot-fix;
- As we discussed and agreed that we are implementing branching model where we always merge towards develop branch and make sufficient tests before merging into master branch and releasing; this PR seems not follow the practice;

I would suggest revisiting this PR after we set up proper CI/CD pipeline, where sufficient test coverage is integrated.

**Comment by KarishmaBothara** [comment]
Created a new PR with respect to develop branch

**Pull request closed without merging**
